test-app/src/androidTest/java/com/avito/android/ui/test/SwipeRefreshTest.kt

```diff
@@ -12,11 +12,32 @@ class SwipeRefreshTest {
     val rule = screenRule<SwipeRefreshActivity>()
 
     @Test
-    fun recyclerView_refresh_onPullToRefresh() {
+    fun refresh_onPullToRefresh_listAction() {
         rule.launchActivity(null)
 
         Screen.swipeRefresh.list.actions.pullToRefresh()
```

Comment:
To be honest, I don't think ListElement is a right place for pullToRefresh action. I do not touch it in order not to cause conflicts